examples/ElmFrontendRocBackend/backend.roc

```diff
@@ -5,15 +5,20 @@ import pf.Task exposing [Task]
 import pf.Http exposing [Request, Response]
 import pf.Utc
 
+# [backend](https://chatgpt.com/share/7ac35a32-dab5-46d0-bb17-9d584469556f) Roc server
+
 main : Request -> Task Response []
 main = \req ->
     # Log request datetime, method and url
     datetime = Utc.now! |> Utc.toIso8601Str
+
     Stdout.line! "$(datetime) $(Http.methodToStr req.method) $(req.url)"
+
     Task.ok {
         status: 200,
         headers: [
+            # TODO check if this header is a good idea
```

Comment:
Is this just used for testing so Elm is happy with resources from localhost? I had a quick google and it wasn't obvious to me. Needs further investigation.

Reply:
Yes. The Elm Frontend wasn't able to load the body of the response from the Roc web server  till I added that header.